server/channels/api4/remote_cluster.go

```diff
@@ -398,7 +393,12 @@ func createRemoteCluster(c *Context, w http.ResponseWriter, r *http.Request) {
 	}
 	rcSaved.Sanitize()
 
-	inviteCode, iErr := c.App.CreateRemoteClusterInvite(rcSaved.RemoteId, url, rcSaved.Token, rcWithTeamAndPassword.Password)
+	password := rcWithTeamAndPassword.Password
+	if password == "" {
+		password = model.NewId()
```

Comment:
There is no need to: the password is used to encrypt the invite code, and we return both as part of the response so the user can send them separately to a remote admin through different channels, and they can be used to accept the invite and establish the connection

Reply:
Ended up going for a random string generated from `crypto/rand` instead of the UUID, PTAL [dde402c](https://github.com/mattermost/mattermost/pull/27864/commits/dde402ce19572bf17c3b371cef1b6e67fd6aa51b)